ajax/libs/bootstrap-star-rating/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "bootstrap-star-rating",
+  "filename": "js/star-rating.min.js",
+  "version": "4.0.1",
+  "author": "Kartik Visweswaran <kartikv2@gmail.com>",
+  "description": "A simple yet powerful JQuery star rating plugin for Bootstrap.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kartik-v/bootstrap-star-rating.git"
+  },
+  "keywords": [
+    "bootstrap",
+    "star",
+    "rating",
+    "font",
+    "awesome",
+    "glyphicon",
+    "svg"
+  ],
+  "license": "BSD-3-Clause",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/kartik-v/bootstrap-star-rating.git",
+    "basePath": "",
+    "files": [
+      "(js|css)/star-rating!(_locale_LANG.js)",
```

Comment:
OK, I have updated, but I'm not sure whether I do correctly or not. Please help me check again, thank you. :D